static/app/components/replays/videoReplayerWithInteractions.tsx

```diff
@@ -118,7 +118,9 @@ export class VideoReplayerWithInteractions {
     if (!hasGestures) {
       // If the replay has no gestures, we should hide the mouse
       // @ts-expect-error private
-      this.replayer.mouse.classList.remove('replayer-mouse');
+      for (const [, {pointerEl}] of Object.entries(this.replayer.pointers)) {
+        pointerEl.classList.remove('replayer-mouse');
+      }
```

Comment:
is this still needed with the new version? don't we only create pointers if it exists in event data?

Reply:
oh you're right, this should be removed